### PR TITLE
Add unsubscribe response

### PR DESCRIPTION
**Pull request by bvdh**
Ticket https://jira.hl7.org/browse/FHIR-40401

https://build.fhir.org/ig/HL7/fhircast-docs/branches/addUnsubscribeResponse/2-4-Subscribing.html

### CODE REVIEW

File: input/pagecontent/2-4-Subscribing.md [hunk 1]
```diff
@@ -162,6 +172,13 @@ Content-Type: application/x-www-form-urlencoded
 
 hub.channel.type=websocket&hub.channel.endpoint=wss%3A%2F%2Fhub.example.com%2Fee30d3b9-1558-464f-a299-cbad6f8135de%0A&hub.mode=unsubscribe&hub.topic=fdb2f928-5546-4f52-87a0-0648e9ded065
 ```
+#### Unscubscribe Response Example
```
Comment (isaacvetter) [inline input/pagecontent/2-4-Subscribing.md:175]:
```suggestion
#### Unsubscribe Response Example

```

File: input/pagecontent/index.md [hunk 2]
```diff
@@ -34,3 +34,20 @@ Note that:
 * [Log issues](https://jira.hl7.org/secure/CreateIssue.jspa), [submit a PR!](https://github.com/fhircast/docs)
 * [Converse at chat.fhir.org](https://chat.fhir.org/#narrow/stream/179271-FHIRcast)
 
+### FHIR Publication Details
```
Comment (isaacvetter) [inline input/pagecontent/index.md:37]: This is largely good, informative content. It's at odds with the tone of the rest of the page, which is a gentle introduction to the functional domain of the specification. 

How about creating a page for this content or identifying another appropriate page?

http://build.fhir.org/ig/HL7/fhircast-docs/branches/addUnsubscribeResponse/#cross-version-analysis

File: input/pagecontent/2-4-Subscribing.md [hunk 3]
```diff
@@ -47,15 +49,15 @@ Content-Type: application/x-www-form-urlencoded
...
+Upon receiving subscription or unsubscription requests, the Hub SHALL respond to a subscription request with an appropriate HTTP response.
 
 If a Hub refuses the request or finds any errors in the subscription request, an appropriate HTTP error response code (4xx or 5xx) SHALL be returned. In the event of an error, the Hub SHOULD return a description of the error in the response body as plain text, to be used by the client developer to understand the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some subscription requests based on their own policies, for example, a Hub may require that all applications subscribe to the same set of events.
 
-#### Subscription Response Example
+In the case of an acceptable subscription requies, an HTTP 202 "Accepted" response is returned. This indicates that the request was received and will now be verified by the Hub. The HTTP body of the response SHALL consist of a JSON object containing an element name of `hub.channel.endpoint` and a value for the WSS URL. The WebSocket WSS URL SHALL be cryptographically random, unique, and unguessable.
```
Comment (isaacvetter) [inline input/pagecontent/2-4-Subscribing.md:58]:
```suggestion
In the case of an acceptable subscription requires, an HTTP 202 "Accepted" response is returned. This indicates that the request was received and will now be verified by the Hub. The HTTP body of the response SHALL consist of a JSON object containing an element name of `hub.channel.endpoint` and a value for the WSS URL. The WebSocket WSS URL SHALL be cryptographically random, unique, and unguessable.

```

File: input/pagecontent/2-4-Subscribing.md [hunk 4]
```diff
@@ -47,15 +49,16 @@ Content-Type: application/x-www-form-urlencoded
...
+Upon receiving subscription or unsubscription requests, the Hub SHALL respond to a subscription request with an appropriate HTTP response.
 
 If a Hub refuses the request or finds any errors in the subscription request, an appropriate HTTP error response code (4xx or 5xx) SHALL be returned. In the event of an error, the Hub SHOULD return a description of the error in the response body as plain text, to be used by the client developer to understand the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some subscription requests based on their own policies, for example, a Hub may require that all applications subscribe to the same set of events.
 
-#### Subscription Response Example
+In the case of an acceptable subscription requires, an HTTP 202 "Accepted" response is returned. This indicates that the request was received and will now be verified by the Hub. The HTTP body of the response SHALL consist of a JSON object containing an element name of `hub.channel.endpoint` and a value for the WSS URL. The WebSocket WSS URL SHALL be cryptographically random, unique, and unguessable.
```
Comment (isaacvetter) [inline input/pagecontent/2-4-Subscribing.md:58]:
```suggestion
In the case of an acceptable subscription request, an HTTP 202 "Accepted" response is returned. This indicates that the request was received and will now be verified by the Hub. The HTTP body of the response SHALL consist of a JSON object containing an element name of `hub.channel.endpoint` and a value for the WSS URL. The WebSocket WSS URL SHALL be cryptographically random, unique, and unguessable.

```

File: input/pagecontent/index.md (hunk 2 above)
Comment (isaacvetter) [reply input/pagecontent/index.md:37]: Discussed on today's FHIRcast. We'll move the Pub Details into the Change Log page, and rename Change Log to Pub Details.

**Comment by bvdh** [comment]
@isaacvetter , @EricOnFHIR please review so I can merge

**Review by EricOnFHIR — APPROVED** [review]

**Pull request merged**